search/include/pcl/search/impl/flann_search.hpp

```diff
@@ -346,11 +345,11 @@ pcl::search::FlannSearch<PointT, FlannDistance>::radiusSearch (
   }
   if (!identity_mapping_)
   {
-    for (size_t j = 0; j < k_indices.size (); ++j )
+    for (auto &k_index : k_indices)
     {
-      for (size_t i = 0; i < k_indices[j].size (); ++i)
+      for (size_t i = 0; i < k_index.size (); ++i)
```

Comment:
What about `for (auto &neighbor_index : k_index)` here?

Reply:
Well, seems clang-tidy don't like nested loops :(. Adjusted.

Reply:
I don't think this was the adjustment @taketwo had in mind. I think he (and me as well) were expecting this line to changed to 
```cpp
for (auto &neighbor_index : k_index)
```
just like it happened in line 219

Reply:
Ahh well, didn't saw this line occurred twice.